docs/CLI/6_reference_types.md

```diff
@@ -0,0 +1,185 @@
+# ORAS Artifact Reference Types (Alpha 1)
+
+## Pushing artifacts that reference other artifacts
+
+The focus on distributing secure supply chain artifacts has driven new innovations for supporting signatures, software bill of materials (SBoMs) and security scan results as artifacts that may be pushed, discovered and pulled, based on the tag or digest reference of a subject artifact.
+
+See the [ORAS Artifacts Spec scenarios][oras-artifacts-scenarios] for more details.
+
+## ORAS Artifact Spec Support
+
+Install the latest alpha build of the oras cli at: https://github.com/oras-project/oras/releases/tag/v0.2.1-alpha.1
+
+## Configure An Oras Artifact Enabled Registry
+
+See [Registry Support](#registry-support)
+
+## Pushing Reference Types
+
+The following walkthrough will generate the following graph of artifacts:
```

Comment:
Reword the double following 
The following walkthrough will generate the graph of artifacts shown below.